src/module_writer.rs

```diff
@@ -656,7 +656,11 @@ pub fn write_bindings_module(
             writer.add_bytes(
                 &module.join("__init__.py"),
                 format!(
-                    "from .{module_name} import *\n\n__doc__ = {module_name}.__doc__\n",
+                    "from .{module_name} import *\n\
+                    \n\
+                    __doc__ = {module_name}.__doc__\n\
+                    if hasattr({module_name}, \"__all__\"):\n\
+                    \t__all__ = {module_name}.__all__\n",
```

Comment:
[Raw string literal](https://stackoverflow.com/questions/26611664/what-is-the-r-operator-in-rust) should be easier.

```suggestion
                    r#"from .{module_name} import *

__doc__ = {module_name}.__doc__
if hasattr({module_name}, "__all__"):
    __all__ = {module_name}.__all__"#,
```

Reply:
I didn't know what your preferred code style is, happy to adjust! 😃